numba/cuda/tests/nocuda/test_function_resolution.py

```diff
@@ -0,0 +1,35 @@
+from numba.cuda.testing import unittest, skip_on_cudasim
+import operator
+from numba.core import types, typing
+
+
+@skip_on_cudasim("Skip on simulator due to use of cuda_target")
+class TestFunctionResolutionNoCuda(unittest.TestCase):
```

Comment:
This test will error if NVVM is not installed, so it needs skipping in that case.

Also, including `NoCuda` in the class name for a test in the `nocuda` module seems redundant.

```suggestion
@unittest.skipIf(not nvvm.is_available(), "No libNVVM")
class TestFunctionResolution(unittest.TestCase):
```